### DISCUSSION TITLE
add ability to see linked projectd id using cli

### DISCUSSION THREAD

**Discussion by x0good** [supabase/supabase#30248] [Feature Requests]
I didn't find how to check current linked projectd id using cli

**Comment by silentworks** [comment]
The CLI doesn't link to projects as far as I can tell, it links to the user and you can view all organisations/projects the user has. Also in the latest version when you are deploying an edge function it rather asks you which project you want to deploy to.

**Reply by x0good** [reply]
but this command link to specific project `supabase link --project-ref $SUPABASE_PROJECT_ID`

and I expect to understand in console to which project previously this command was executed
